python/rikai/spark/sql/generated/RikaiModelSchemaLexer.py

```diff
@@ -1,9 +1,9 @@
 # Generated from src/main/antlr4/org/apache/spark/sql/ml/parser/RikaiModelSchema.g4 by ANTLR 4.9.1
-from antlr4 import *
+import sys
```

Comment:
Can we set `isort` to ignore this directory?

Those changes will be override next time we generate the code.